libraries/plugins/webserver/webserver_plugin.cpp

```diff
@@ -34,6 +33,11 @@ using boost::asio::ip::tcp;
 using std::shared_ptr;
 using websocketpp::connection_hdl;
 
+#define WEBSERVER_DEFAULT_THREAD_POOL_SIZE 256
+#define WEBSERVER_THREAD_POOL_SIZE_OPTION "webserver-thread-pool-size"
```

Comment:
Mainly because we do not want to be in the habit of change config option names. I like how the default value is set up.